src/model/user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({ name: 'user' })
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ length: 100 })
+  name: string;
+
+  @Column({ length: 255 })
+  email: string;
```

Comment:
Make the email as the unique (primary) field.

Reply:
done